src/main/twirl/views/showOpportunity.scala.html

```diff
@@ -29,65 +29,132 @@
 }
 
 @main(s"Opportunity ${opportunity.id.id}") {
-    <div class="grid-row">
-        <a href="@controllers.routes.OpportunityController.showOpportunities()">&lt; Return to opportunity list</a>
-        <p></p>
-        <h1>@opportunity.title</h1>
+<div class="grid-row">
 
+    <div class="column-full">
+        <p>
```

Comment:
I recommend using the breadcrumbs pattern for this (c.f. the top of the application pages)